src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -117,76 +117,79 @@ export const RegistrationSearchBar = ({ registrationQuery }: Pick<SearchPageProp
-                {values.properties.map((property, index) => (
-                  <AdvancedSearchRow
-                    key={index}
-                    removeFilter={() => {
-                      remove(index);
-                      const valueToRemove = typeof property.value === 'string' ? property.value : property.value[0];
-                      const newParams = removeSearchParamValue(searchParams, property.fieldName, valueToRemove);
-                      newParams.set(ResultParam.From, '0');
-                      history.push({ search: newParams.toString() });
-                    }}
-                    baseFieldName={`properties[${index}]`}
-                  />
-                ))}
-
-                <Box sx={{ display: 'flex', gap: '1rem' }}>
-                  <Button
-                    data-testid={dataTestId.startPage.advancedSearch.addFilterButton}
-                    variant="outlined"
-                    onClick={() => {
-                      const newPropertyFilter: PropertySearch = {
-                        fieldName: '',
-                        value: '',
-                      };
-                      push(newPropertyFilter);
-                    }}
-                    startIcon={<FilterAltIcon />}>
-                    {t('search.add_filter')}
-                  </Button>
+              <>
+                <Box gridArea="filter" sx={{ display: 'flex', flexDirection: 'column', gap: '0.75rem' }}>
+                  {values.properties.map((property, index) => (
+                    <AdvancedSearchRow
+                      key={index}
+                      removeFilter={() => {
+                        remove(index);
+                        const valueToRemove = typeof property.value === 'string' ? property.value : property.value[0];
+                        const newParams = removeSearchParamValue(searchParams, property.fieldName, valueToRemove);
+                        newParams.set(ResultParam.From, '0');
+                        history.push({ search: newParams.toString() });
+                      }}
+                      baseFieldName={`properties[${index}]`}
+                    />
+                  ))}
+                </Box>
+                <Box gridArea="buttonRowBottom" sx={{ display: 'flex', alignItems: 'flex-end' }}>
                   {values.properties && values.properties.length > 0 && (
-                    <Button
-                      variant="contained"
-                      type="submit"
-                      startIcon={<SearchIcon />}
-                      data-testid={dataTestId.startPage.advancedSearch.searchButton}>
-                      {t('common.search')}
-                    </Button>
+                    <>
+                      <IconButton
+                        sx={{ mr: '0.5rem', borderRadius: '4px', minWidth: '36px', minHeight: '36px' }}
+                        size="small"
```

Comment:
Mener knappene hadde primary color før, bevisst at de nå er grå, eller en glipp?
```suggestion
                        size="small"
                        color="primary"
```